runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -55,7 +55,7 @@ processResources {
     'dataflow.legacy_environment_major_version' : '8',
     'dataflow.fnapi_environment_major_version' : '8',
     'dataflow.legacy_container_version' : 'beam-master-20211213',
-    'dataflow.fnapi_container_version' : 'beam-master-20210913',
+    'dataflow.fnapi_container_version' : 'beam-master-20210119',
```

Comment:
@emilymye Should this be beam-master-20220119 (2022, not 2021)? I'm not sure how to validate the value myself, just basing my comment on the fact it looks like a date and all previous changes have been increases.

Reply:
you are right! Thanks for the catch